src/components/CityWeatherCard/CityWeatherCard.module.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
Understand that this is being broken down into multiple PRs although ordinarily I would probably either ask here that this file should be doing something to be committed